src/EnergyPlus/CurveManager.hh

```diff
@@ -571,6 +572,32 @@ namespace CurveManager {
 	int
 	GetCurveObjectTypeNum( int const CurveIndex ); // index of curve in curve array
 
+	int
+	GetCurveInterpolationMethodNum( int const CurveIndex ); // index of curve in curve array
+
+	void
+	ReadTableDataFromFile(
```

Comment:
There is an existing function named `ReadTableData` that is used by Table:MultiVariableLookup and this new function is named `ReadTableDataFromFile`.  Since this one is specific to Table:TwoIndependentVariables, it should be renamed.  Maybe `ReadTwoVarTableData` or `ReadTwoVarTableDataFromFile` .